db/migrations/1699388449731-Data.js

```diff
@@ -0,0 +1,87 @@
+module.exports = class Data1699388449731 {
+    name = 'Data1699388449731'
+
+    async up(db) {
```

Comment:
make migrations file pretty like it used to be

Reply:
`prettier`?